fn test() {}
fn test() {}
#! ERROR already have a function named `test`